src/styles/blocks/page.scss

```diff
@@ -0,0 +1,19 @@
+.page {
+  margin: 0;
+  padding-top: 0;
+}
+
+.page-body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $container-size;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
set it for `body` or `html`
use container only for positioning if you really need